### DISCUSSION TITLE
Copy & Paste from bigger to smaller image

### DISCUSSION THREAD

**Discussion by rottmann** [Orama-Interactive/Pixelorama#727] [Q&A] [answered]
Copy a selection from a bigger image into a smaller is not possible, if the selection on the source is outside the size of the target image.

On paste, it should be pasted into the left top corner and could then be moved.

**Comment by rottmann** [comment]
Closed. 
[Same in #721](https://github.com/Orama-Interactive/Pixelorama/discussions/721)

**Comment by Variable-ind** [comment]
Pasting from large to small image is possible.
You just have to zoom_out a little to find it

**Comment by OverloadedOrama** [comment, marked as answer]
I changed the paste behavior recently and the change will be included in the next version, see https://github.com/Orama-Interactive/Pixelorama/discussions/721#discussioncomment-3648549
